### DISCUSSION TITLE
How to fix AttributeError: module has no attribute

### DISCUSSION THREAD

**Discussion by HiverXD** [community/community#138641] [Programming Help] [answered]
### Body

I just installed anaconda, and jupyter notebook. And was testing its features. 
Then, I found a problem while importing the module, and I haven't found a solution yet. 

I tried making a simplified code, but the same problem still occurs. 
Please help. 

![image](https://github.com/user-attachments/assets/e71faffc-91ae-404a-8106-4cc31ff5f8f1)

The test.py file was written as above.

![image](https://github.com/user-attachments/assets/c5b4ee6a-7630-4b9e-a878-d4edf99788b2)

The test_2.ipynb file was written as above.
When I run this program, an error occurs.

![image](https://github.com/user-attachments/assets/36bebcf6-c38d-4ac1-9bed-4e92833b761c)

Clearly, these files are in the same directory.
Why is this happening?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by jgschmitz** [comment, marked as answer]
The issue with the error "module has no attribute 'addition'" likely arises because Python may be trying to import the script incorrectly due to naming conflicts or caching issues in Jupyter notebooks. Here are the steps to resolve this:

Rename the test.py file to something other than test (e.g., my_test.py). This avoids conflicts with Python's internal test module.
Restart the Jupyter kernel after renaming and re-run the notebook.

**Reply by HiverXD** [reply]
Thank you for your help !! !!!

**Comment by Richard-Gidi** [comment]
Try using a different name instead of "test" as there is a standard library module named "test" that is mainly used for internal testing of Python itself, and it is not meant for general use.

**Reply by HiverXD** [reply]
thank you. I've solved that problem by changing name of the file.
